Platform/ElkhartlakeBoardPkg/BoardConfig.py

```diff
@@ -252,34 +252,36 @@ def GetContainerList (self):
           ('IPFW',      'SIIPFW.bin',          '',     container_list_auth_type,   'KEY_ID_CONTAINER'+'_'+self._RSA_SIGN_TYPE,    0,           0        ,      0),   # Container Header
         )
 
+        bins = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'Binaries')
+
         if self.ENABLE_TCC_TUNING:
             container_list.append (
-              ('TCCT','TccTuning.bin',      'Lz4',     container_list_auth_type,   'KEY_ID_CONTAINER_COMP'+'_'+self._RSA_SIGN_TYPE, 0,   self.TCCT_SIZE,      0),   # TCC Tuning
+              ('TCCT','TccTuning.bin' if os.path.exists(os.path.join(bins, 'TccTuning.bin')) else '',      'Lz4',     container_list_auth_type,   'KEY_ID_CONTAINER_COMP'+'_'+self._RSA_SIGN_TYPE, 0,   self.TCCT_SIZE,      0),   # TCC Tuning

```

Comment:
Had better use a variable for component name to avoid so long in one line. e.g.
 CompFile = 'TccTuning.bin' if os.path.exists(os.path.join(bins, 'TccTuning.bin')) else ''
 ('TCCT',    CompFile,      'Lz4', ...

Reply:
Done updating it.